manifest.versions.json

```diff
@@ -34,7 +34,7 @@
       "kb|ltsc2019|default": "$(kb|ltsc2019|4.7.2)",
       "lcu|ltsc2019|default": "$(lcu|ltsc2019|4.7.2)",
 
-      "nuget|version": "6.8.0",
+      "nuget|version": "6.9.1",
```

Comment:
Because of this change, all the sdk Dockerfiles will be changed. Which means we'll need the datestamp applied to tags for all of them.